### PR TITLE
Rewrite doc with named-export/import

### LINKED ISSUE #3165: Use the named export instead of the default export at the Customize Theme Page
### Provide the exact quote of the error or issue

Use the named export where it uses the default export at the Customize Theme Page. For example...
from: 
```ts title=./theme/index.ts
import { extendTheme, UsageTheme } from "@yamada-ui/react"
// import styles from './styles'
// import components from './components'
// import tokens from './tokens'

const customTheme: UsageTheme = {
  // styles,
  // components,
  // ...tokens,
}

export default extendTheme(customTheme)()
```
to: 
```ts title=./theme/index.ts
import { extendTheme, UsageTheme } from "@yamada-ui/react"

export const customTheme: UsageTheme = {
  // styles,
  // components,
  // ...tokens,
}

export const theme = extendTheme(customTheme)()
```

### Does the docs page already exist? Please link to it.

https://yamada-ui.com/styled-system/theming/customize-theme

### Are you willing to participate in fixing this issue and create a pull request with the fix?

None

### PR DESCRIPTION
**Pull request by Shunii85**
Closes #3165 

## Description
Rewrite with named-export/import instead of default-export/import

### CODE REVIEW

**Review by aiirononeko — DISMISSED** [review]
LGTM!

**Review by hirotomoyamada — CHANGES_REQUESTED** [review]
The `customize-theme.ja.mdx`'s Lines 34, 35, 36, 523, and 524 have not been corrected.

The `customize-theme.mdx`'s Line 419 have not been corrected.

File: docs/contents/styled-system/theming/customize-theme.ja.mdx
Comment (hirotomoyamada) [inline docs/contents/styled-system/theming/customize-theme.ja.mdx:1]: Lines 34, 35, 36, 523, and 524 have not been corrected.

File: docs/contents/styled-system/theming/customize-theme.mdx
Comment (hirotomoyamada) [inline docs/contents/styled-system/theming/customize-theme.mdx:1]: Line 419 have not been corrected.

**Review by hirotomoyamada — CHANGES_REQUESTED** [review]
- `docs/contents/styled-system/theming/component-styles.ja.mdx`
- `docs/contents/styled-system/theming/component-styles.mdx`
- `docs/contents/styled-system/animation.ja.mdx`
- `docs/contents/styled-system/animation.mdx`

These files also need to be modified.

**Comment by Shunii85** [comment]
@hirotomoyamada I have done! Please review.

**Review by hirotomoyamada — APPROVED** [review]

**Pull request merged**